src/libraries/Common/tests/WasmTestRunner/WasmTestRunner.cs

```diff
@@ -57,6 +59,10 @@ public static async Task<int> Main(string[] args)
                 case "-untilFailed":
                     untilFailed = true;
                     break;
+                case "-verbosity":
```

Comment:
From 2 new args we care about parsing "verbosity" only, ignoring "debug"?

Reply:
It's value pair